package.json

```diff
@@ -1,56 +1,44 @@
 {
-  "name": "QualityNinja",
-  "version": "0.1.6",
+  "name": "twitch-flipper",
+  "version": "0.0.1",
   "description": "",
-  "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 0",
-    "build": "npm run build-partial:core && npm run build-partial:popup",
-    "build:production": "bash scripts/build_static.sh production",
-    "build-partial:core": "bash scripts/build_static.sh core",
-    "build-partial:popup": "bash scripts/build_static.sh popup",
-    "lint": "eslint . -c .eslintrc.json --ext js,jsx"
+    "build": "./scripts/build.sh",
+    "start": "react-scripts start"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/dan-lovelace/quality-ninja.git"
   },
   "keywords": [],
-  "author": "Dan Lovelace",
+  "author": "",
   "license": "ISC",
   "bugs": {
     "url": "https://github.com/dan-lovelace/quality-ninja/issues"
```

Comment:
should this be updated to this repo's issue page?

Reply:
Yes, updated.